base/abstractarray.jl

```diff
@@ -1982,11 +1982,12 @@ function hvcat(nbc::Integer, as...)
 end
 
 """
-    hvcat(rows::Tuple{Vararg{Int}}, values...)
+    hvcat(blocks_per_row::Union{Tuple{Vararg{Int}}, Integer}, values...)
```

Comment:
should this be `Tuple{Vararg{Integer}}` instead?

Reply:
The signatures below indicate it must be `Int`. Also, this fails

```julia
hvcat((Int8(2),Int8(2),Int8(2)), a,b,c,d,e,f)
```

Reply:
Ah perhaps the single integer argument should also be an `Int` in that case, as this fails as well:
```julia
julia> hvcat(Int8(2), 1:6...)
ERROR: MethodError: no method matching hvcat(::Tuple{Int8, Int8, Int8}, ::Int64, ::Int64, ::Int64, ::Int64, ::Int64, ::Int64)
Closest candidates are:
  hvcat(::Integer, ::Any...) at abstractarray.jl:1942
  hvcat(::Tuple{Vararg{Int64}}, ::T...) where T<:Number at abstractarray.jl:2038
  hvcat(::Tuple{Vararg{Int64}}, ::Number...) at abstractarray.jl:2075
  ...
Stacktrace:
 [1] hvcat(::Int8, ::Int64, ::Vararg{Int64})
   @ Base ./abstractarray.jl:1948
 [2] top-level scope
   @ REPL[2]:1
```